src/db_api_util.ts

```diff
@@ -591,6 +756,22 @@ export async function logVolunteerVoterClaimToDb(
   }
 }
 
+export async function getVoterHasVolunteer(
+  userId: string,
+  userPhoneNumber: string
+): Promise<boolean> {
+  const client = await pool.connect();
+  try {
+    const result = await client.query(
+      'SELECT * FROM volunteer_voter_claims WHERE user_id = $1 AND user_phone_number = $2',
```

Comment:
This is a bit faster: `SELECT EXISTS(SELECT 1 FROM volunteer_voter_claims WHERE user_id = $1 AND user_phone_number = $2))` (because it's not reading any values, it can figure out the answer straight from the index without having to load in the pages of data)